Happy Dance/Assets/_Scripts/EnableGirl.cs

```diff
@@ -15,28 +15,34 @@ public class EnableGirl : MonoBehaviour
 
         private void Start()
         {
-
-        objectToActivate.SetActive(false);
-        StartCoroutine(ActivationRoutine());
-
-		//audioSource = mainCamera.Find.GetComponent<AudioSource>();
-
-		//AudioSource.PlayClipAtPoint(beam, new Vector3(5, 1, 2));
-
+            if (objectToActivate != null)
+            {
```

Comment:
Braces here are a _should_, but in practice it seems we're settling on simple conditionals like these to not use braces. If only to fit more code onto a page.